webpack.prod.js

```diff
@@ -0,0 +1,66 @@
+const path = require('path');
+const merge = require('webpack-merge');
+const webpack = require('webpack');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+const CompressionPlugin = require('compression-webpack-plugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const baseConfig = require('./webpack.base.js');
+
+module.exports = merge(baseConfig, {
```

Comment:
I'm not convinced with your webpack set up.

A whole lot of things you are doing have been changed with better set up.

webpack 4 has an `optimization` object which you should modify and work with.


also the `bail` object to terminate early on error

Reply:
Feedback Implemented, thanks